docs/internals/contributing.rst

```diff
@@ -51,7 +51,7 @@ For Windows the use of cmd under Administrator permission is suggested instead o
 
 .. code-block:: batch
 
-    > virtualenv -p "C:\Python34\python.exe" env
+    > C:\Python34\python.exe -m venv env
```

Comment:
I agree that this is a problem. I was planning to make another pull request after this one to remove that full path and use the built-in `py -3` that comes when install Python on Windows.